File: php/examples/post_services.php
<?php
require_once 'base_request.php';

// Example of POST /services

$debug = true;
$company_id = $_ENV['COMPANY_ID'];
$jurisdiction_id = $_ENV['JURISDICTION_ID'];

$jurisdiction_ids = array($jurisdiction_id);

$request_data = array(
    'company_id' => $company_id,
    'jurisdiction_ids' => $jurisdiction_ids
);

$request_params = '';

$json_body = json_encode($request_data);

send_request('POST', '/services', $request_params, $json_body);

File: csharp/examples/PostServices.cs
using System;
namespace Examples.examples.models
{
    // Example of POST /services
	public class PostServices : BaseRequest
	{
        private string _companyId;
        private string _companyName;
        private string _jurisdictionId;

        public PostServices()
		{
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            _companyName = Environment.GetEnvironmentVariable("COMPANY_NAME");
            _jurisdictionId = Environment.GetEnvironmentVariable("JURISDICTION_ID");
            Console.WriteLine($"PostServices: _companyId={_companyId} _companyName={_companyName} _jurisdictionId={_jurisdictionId}");
        }

        public override void SendRequest()
        {
            string[] jurisdiction_ids = new string[] { _jurisdictionId };
            var body = Newtonsoft.Json.JsonConvert.SerializeObject(new
            {
                company_id = _companyId,
                //company = _companyName,
                jurisdiction_ids = jurisdiction_ids
            });

            PostRequest("services", body);
        }
    }
}

